src/components/constants/Pokemon.ts

```diff
@@ -200,8 +240,15 @@ export class Pokemon {
             "info": [
                 "Using its ability to read minds, it will identify impending danger and TELEPORT to safety."
             ],
-            "evolvesInto": "Kadabra",
-            "evolvesVia": "Level 16",
+            "evolution": [{
```

Comment:
Let's leave these out in this PR, then add them in a separate one. It'll make it easier to review. No need for you to do this work repeatedly until the format is resolved.